config/lava/iec-security/generic-qemu-tftp-ramdisk-iec-security-template.jinja2

```diff
@@ -0,0 +1,11 @@
+{% extends 'boot/generic-qemu-boot-template.jinja2' %}
```

Comment:
It will be interesting to see if this file name works:
`config/lava/iec-security/generic-qemu-tftp-ramdisk-iec-security-template.jinja2`
as in principle the file name for QEMU should not include `tftp` and `ramdisk`.  Let's check the results on staging once this has been deployed.

Reply:
Actually this PR is not enabling the test plan on QEMU.

Reply:
added qemu-x86-64